react/components/FacetCheckboxList.js

```diff
@@ -1,17 +1,20 @@
-import classNames from 'classnames'
-import React, { useContext, useState, useMemo } from 'react'
-import { Checkbox } from 'vtex.styleguide'
-import { applyModifiers } from 'vtex.css-handles'
-import { useRuntime } from 'vtex.render-runtime'
+import classNames from "classnames";
```

Comment:
Lint this file. Go to the root of the directory and run `yarn lint`

Reply:
I lint all files.